### ISSUE TITLE
Warning with numpy 1.17rc1

### ISSUE THREAD

**Issue by jni** [napari/napari#404]
## 🐛 Bug

Running `examples/add_image.py` with NumPy 1.17rc1 results in the following warning:

```python
 $ python examples/add_image.py 
/home/jni/miniconda3/envs/cf/lib/python3.6/site-packages/vispy/visuals/line/line.py:395: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  ('color', np.float32, 4)])
/home/jni/miniconda3/envs/cf/lib/python3.6/site-packages/vispy/visuals/line/arrow.py:57: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  ('linewidth', np.float32, 1)
```

Coming from within vispy. @djhoese is this fixed on master / easy to fix?

**Comment by djhoese** [comment]
No this is not. Here's the full code for the `line.py` portion:

```
    _agg_vtype = np.dtype([('a_position', np.float32, 2),
                           ('a_tangents', np.float32, 4),
                           ('a_segment',  np.float32, 2),
                           ('a_angles',   np.float32, 2),
                           ('a_texcoord', np.float32, 2),
                           ('alength', np.float32, 1),
                           ('color', np.float32, 4)])
```

What is the new method that we should be using?

**Comment by jni** [comment]
Great question! Let's ask someone here at the conference! =P

@mattip perhaps?

**Comment by mattip** [comment]
According to the message, you should add parenthesis around the shape parameter (or eliminate it if 1:
```
_agg_vtype = np.dtype([('a_position', np.float32, (2,)),
                           ('a_tangents', np.float32, (4,)),
                           ('a_segment',  np.float32, (2,)),
                           ('a_angles',   np.float32, (2,)),
                           ('a_texcoord', np.float32, (2,)),
                           ('alength', np.float32),
                           ('color', np.float32, (4,))])
```
Does this work, and if so can you suggest an alternative wording for the warning?

**Comment by djhoese** [comment]
Talked to @seberg at SciPy and he explained it to me. The above should work. I might make a PR for numpy to improve the warning message.

**Comment by djhoese** [comment]
Fixed in #1662, will be included in vispy 0.6. Feel free to close.

**Comment by jni** [comment]
Awesome! Gotta love this conference! :grinning:
